src/ImageSharp/ImageSharp.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.SourceLink.GitHub"/>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" />
```

Comment:
Dp we not need `PrivateAssets="All"` here?

Reply:
It's already marked private in the `Directory.Build.targets` file.